lib/Normalize_std_ast.ml

```diff
@@ -209,27 +148,44 @@ let make_mapper conf ~ignore_doc_comments ~erase_jane_syntax =
                         , [] ) } ] }
     | _ -> Ast_mapper.default_mapper.attribute m attr
   in
-  (* sort attributes *)
-  let attributes (m : Ast_mapper.mapper) (atrs : attribute list) =
+  let map_attributes_no_sort (m : Ast_mapper.mapper) (atrs : attribute list)
+      =
     let atrs =
       if erase_jane_syntax then
         List.filter atrs ~f:(fun a -> not (is_erasable_jane_syntax a))
-      else atrs
+      else
+        List.filter atrs ~f:(fun a ->
+            not (is_erasable_builtin_jane_syntax a) )
     in
     let atrs =
       if ignore_doc_comments then
         List.filter atrs ~f:(fun a -> not (is_doc a))
       else atrs
     in
-    Ast_mapper.default_mapper.attributes m atrs |> sort_attributes
+    Ast_mapper.default_mapper.attributes m atrs
+  in
+  let attributes (m : Ast_mapper.mapper) (atrs : attribute list) =
+    sort_attributes (map_attributes_no_sort m atrs)
   in
   let expr (m : Ast_mapper.mapper) exp =
-    let exp = {exp with pexp_loc_stack= []} in
+    let exp =
+      { exp with
+        pexp_loc_stack= []
+      ; pexp_attributes= map_attributes_no_sort m exp.pexp_attributes }
```

Comment:
After we're done killing off jane-syntax, this can be removed. Maybe leave a comment? I would start it `(* CR jane-syntax: ... *)`. This is different from the convention we use internally in two ways:

1. We actually commit code with `CR`. So `CR` really means something like `CR-someday`.
2. We label with the project, not the person. This makes these items easier to find later.

Feel free to add more such comments if there are other items to return to.

Reply:
refactored this a bit, but added comments everywhere necessary